examples/kyb-app/src/hooks/useActiveWorkflowQuery/useActiveWorkflowQuery.ts

```diff
@@ -1,96 +1,18 @@
-import {
-  DocumentConfiguration,
-  FlowData,
-  collectionFlowQuerykeys,
-} from '@app/domains/collection-flow';
-import { fetchFile } from '@app/domains/storage/storage.api';
+import { DocumentConfiguration, collectionFlowQuerykeys } from '@app/domains/collection-flow';
 import { useSessionQuery } from '@app/hooks/useSessionQuery';
-import { collectionFlowFileStorage } from '@app/pages/CollectionFlow/collection-flow.file-storage';
-import { traverseObjectAndPickValue } from '@app/utils/traverse-object-and-pick-value';
-import { traverseObjectAndSetValue } from '@app/utils/traverse-object-and-set-value';
 import { useQuery } from '@tanstack/react-query';
-import { useEffect, useRef, useState } from 'react';
 
 export const useActiveWorkflowQuery = (documentConfigurations: DocumentConfiguration[]) => {
   const { user } = useSessionQuery();
-  const [isLoading, setLoading] = useState(false);
-  const { data: _flowData, isFetching } = useQuery({
-    ...collectionFlowQuerykeys.getFlowData({ endUserId: user?.id }),
+  const { data: flowData = null, isFetching } = useQuery({
+    ...collectionFlowQuerykeys.getFlowData({ endUserId: user?.id, documentConfigurations }),
```

Comment:
It seems some places use the word `get` and some `fetch`. If it's an async HTTP request it should be `fetch`. :)